plugins/org.jkiss.dbeaver.ext.dm.ui/src/org/jkiss/dbeaver/ext/dm/ui/views/DMUIMessages.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jkiss.dbeaver.ext.dm.ui.views;
+
+import org.eclipse.osgi.util.NLS;
+
+/**
+ * @author Shengkai Bai
+ */
+public class DMUIMessages extends NLS {
+    static final String BUNDLE_NAME = "org.jkiss.dbeaver.ext.mssql.ui.DMUIMessages";
+
+    public static String dialog_connection_host_label;
+
+    public static String dialog_connection_port_label;
+
+    public static String dialog_connection_database_schema_label;
```

Comment:
I can not find places where these (schema/user/password) rows are used in the code. 
Do the Dameng connection needed in the database or schema field?